analyzer/windows/tests/test_analyzer.py

```diff
@@ -684,3 +692,351 @@ def test_choose_package_zip_compound(self):
         self.assertEqual("modules.packages.zip_compound", pkg_name)
         self.assertEqual(pkg_class.__class__.__name__, "ZipCompound")
 
+
+class TestAnalyzerMonitoring(unittest.TestCase):
+    def setUp(self):
+        self.patches = []
+        patch_sleep = patch("lib.common.defines.KERNEL32.Sleep")
+        _ = patch_sleep.start()
+        self.addCleanup(patch_sleep.stop)
+        patch_call = patch("subprocess.call")
+        self.call = patch_call.start()
+        self.addCleanup(patch_call.stop)
+        self.analyzer = Analyzer()
+        self.cph = CommandPipeHandler(self.analyzer)
```

Comment:
I think we use "pipe_handler" in the analyzer now, or maybe "pipe_hndlr" - cph is a bit too obtuse